tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Response.cs

```diff
@@ -15,26 +15,35 @@ public class Response
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public List<string> ResponseErrors { get; set; }
 
+    [JsonPropertyName("next_steps")]
```

Comment:
```suggestion
    [JsonPropertyName("nextStep")]
```

Reply:
Our mcp tool names are snake case so I'm inclined to say all json properties also be snake case